helloworld.java

```diff
@@ -0,0 +1,3 @@
+public class a{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class names should begin with an uppercase character](https://app.codacy.com/app/gagsdeep/TestSonar/pullRequest?prid=4663122)